protocol/messenger_contacts.go

```diff
@@ -887,14 +887,9 @@ func (m *Messenger) BlockContact(contactID string, fromSyncing bool) (*Messenger
 	//		 This should be considered separately, I'm not sure if that's safe.
 	//		 https://github.com/status-im/status-go/issues/3720
 	if !fromSyncing {
-		notifications, err := m.persistence.DismissAllActivityCenterNotificationsFromUser(contactID, m.getCurrentTimeInMillis())
-		if err != nil {
-			return nil, err
-		}
-
-		err = m.syncActivityCenterNotifications(notifications)
```

Comment:
I don't remember this very good, but I think `fromSyncing` flag is not needed anymore if we remove `syncActivityCenterNotifications` from here.  

Ping me to discuss this if needed.

Reply:
Also, shouldn't `dismiss` action be synced from here?